components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRManagementService.java

```diff
@@ -207,6 +207,8 @@ private RegistrationResponseProfile createOAuthApplication(RegistrationRequestPr
                 log.debug("Creating OAuth App " + applicationName);
             }
 
+            oAuthConsumerApp.setBackChannelLogoutUrl(profile.getBackchannelLogoutUri());
+
```

Comment:
Shall we move this before line 206 so it is consistent and debug log is printed when the app is to be created